vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -125,6 +125,7 @@ public String toString() {
     public List<Category> categories;
     @Nullable
     private String password;
+    private String resourceKey;
```

Comment:
It might be nice to add a JavaDoc to this field just so consumers of the library will get why we have it (since it might be useful to them).